### PR TITLE
docs: optimize the config documents

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnopg@gmail.com>

### Ⅱ. Does this pull request fix one issue?

ref #1094 

### Ⅴ. Special notes for reviews

I delete the configurations that unused for now.

### CODE REVIEW

File: docs/config/dfdaemon_config_template.yml [hunk 1]
```diff
@@ -49,28 +58,14 @@ dfget_flags: ["--node","192.168.33.21","--verbose","--ip","192.168.33.23",
...
-#Open detail info switch
-verbose: false
+# dfget bin path
+# default: the relative file path for the dfdaemon
+# dfpath: ./dfget
 
-# The maximum number of CPUs that the dfdaemon can use
-maxprocs: 10
+# Verbose indicates whether to be verbose.
+# If set true, log level will be 'debug'.
```
Comment (yeya24) [inline docs/config/dfdaemon_config_template.yml]: Verbose mode. If true, set log level to 'debug'.

File: docs/config/dfdaemon_properties.md [hunk 2]
```diff
@@ -9,15 +9,12 @@ The following startup parameters are supported for `dfdaemon`
...
+| dfpath | dfget bin path |
 | hijack_https | HijackHTTPS is the list of hosts whose https requests should be hijacked by dfdaemon. The first matched rule will be used |
 | localrepo | Temp output dir of dfdaemon, by default `$HOME/.small-dragonfly/dfdaemon/data/` |
-| maxprocs| The maximum number of CPUs that the dfdaemon can use |
 | proxies | Proxies is the list of rules for the transparent proxy |
-| ratelimit | Net speed limit,format:xxxM/K |
 | registry_mirror | Registry mirror settings |
-| supernodes | Specify the addresses(host:port) of supernodes, it is just to be compatible with previous versions |
-| verbose | Open detail info switch |
+| verbose | Verbose indicates whether to be verbose. If set true, log level will be `debug`. |
```
Comment (yeya24) [inline docs/config/dfdaemon_properties.md]: ditto

File: docs/config/dfdaemon_config_template.yml [hunk 3]
```diff
@@ -49,28 +58,14 @@ dfget_flags: ["--node","192.168.33.21","--verbose","--ip","192.168.33.23",
...
-dfpath: ./dfget
-
-# https options
-# port: 12001
-# hostIp: 127.0.0.1
-# certpem: ""
-# keypem: ""
+# the temp output dir of dfget, it must be an absolute path.
+# default: `$HOME/.small-dragonfly/dfdaemon/data/`
+# localrepo: $HOME/.small-dragonfly/dfdaemon/data/
```
Comment (yunfeiyanggzq) [inline docs/config/dfdaemon_config_template.yml]: emmm.`# default: `$HOME/.small-dragonfly/dfdaemon/data/`` look like the configuration which is annotated,how about 
```
the temp output dir of dfget, it must be an absolute path.the default value is $HOME/.small-dragonfly/dfdaemon/data/
# localrepo: $HOME/.small-dragonfly/dfdaemon/data/
```

File: docs/config/dfdaemon_config_template.yml [hunk 4]
```diff
@@ -1,42 +1,51 @@
 # This file is the template of dfdaemon configuration file.
 # You can configure your dfdaemon by change the parameter according your requirement.
 
-# RegistryMirror configures the mirror of the official docker registry
```
Comment (zhouhaibing089) [inline docs/config/dfdaemon_config_template.yml]: Can we add yamllint in CI to ensure that all the yaml files are correctly formatted?

Comment (allencloud) [reply docs/config/dfdaemon_config_template.yml]: Is it https://github.com/adrienverge/yamllint?

Comment (zhouhaibing089) [reply docs/config/dfdaemon_config_template.yml]: Right, that's the one!

**Comment by Starnop** [comment]
> I think that it's better to use default value in the template files because that maybe most people will use the template file for testing.

We can do that with PR #1084. also cc/ @SataQiu

File: docs/config/dfget_config_template.yml [hunk 5]
```diff
@@ -14,11 +14,15 @@ localLimit: 20M
...
 
-# TotalLimit rate limit about the whole host, format: G(B)/g/M(B)/m/K(B)/k/B
+# TotalLimit rate limit about the whole host includes download and upload, format: G(B)/g/M(B)/m/K(B)/k/B
 # pure number will also be parsed as Byte.
-totalLimit: 40M
+#
+# Suppose that there are two tasks on the same host
+#  and the `localLimit` for each task is 20MB.
+#  The actual download speed limit for each task will be 10MB when the `totalLimit` is 20MB.
+totalLimit: 20M
```
Comment (lowzj) [inline docs/config/dfget_config_template.yml]: comment this line, user can enable it

File: docs/config/dfget_config_template.yml [hunk 6]
```diff
@@ -14,11 +14,15 @@ localLimit: 20M
 
 # Minimal rate about a single download task, format: G(B)/g/M(B)/m/K(B)/k/B
 # pure number will also be parsed as Byte.
-minRate: 512
+minRate: 512B
```
Comment (lowzj) [inline docs/config/dfget_config_template.yml]: set the value as same as the value in the code

File: docs/config/dfget_config_template.yml (hunk 5 above)
Comment (Starnop) [reply docs/config/dfget_config_template.yml]: DONE.

File: docs/config/dfget_config_template.yml (hunk 6 above)
Comment (Starnop) [reply docs/config/dfget_config_template.yml]: DONE.

**Pull request merged**